pkg/operator/operator.go

```diff
@@ -180,6 +191,20 @@ func (o *Operator) Start(stop <-chan struct{}) error {
 		log.Info("cache synced")
 	}
 
+	// Sync the CA cert every once in a while.
+	ticker := time.NewTicker(24 * time.Hour)
```

Comment:
Once should be enough, unless there are errors, and so many errors that we hit the max retry limit, but if that ends up being a real problem we could revisit how else to edge trigger or otherwise deal with the period and/or retry mechanics.